src/offset_parser.rs

```diff
@@ -0,0 +1,22 @@
+// #![allow(warnings)]
+use pulldown_cmark::{Event};
+
+pub struct OffsetParser<'a>(pub ::pulldown_cmark::Parser<'a>);
+
+impl<'a> Iterator for OffsetParser<'a> {
+    type Item = (Event<'a>, usize);
+
+    fn next(&mut self) -> Option<(Event<'a>, usize)> {
+        loop {
+            let &mut OffsetParser(ref mut inner_iter) = self;
+            let offset = inner_iter.get_offset();
```

Comment:
Re <https://github.com/killercup/rust-docstrings/pull/1/files#r96317886>: Oh, I see: pulldown only gives us a usize. Any chance we can get the size here to calculate the whole span?